src/elements/content-sharing/ContentSharing.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * @flow
```

Comment:
Can this be written in TypeScript? We're going to struggle to fully convert if we continue to write new components using Flow.

Reply:
Unfortunately, no, because of the dependencies on Flow types (src/common/types/core.js, src/common/types/api.js, etc.)